### DISCUSSION TITLE
Vaccination data for UK and Australia

### DISCUSSION THREAD

**Discussion by ariannecaritos-kinesso** [owid/covid-19-data#2741] [Q&A] [answered]
Hi! Would just like to know why some vaccination data in Australia and UK has stopped in September? Just want to confirm if this will be a thing going forward or if this is just temporary. Thanks!

**Comment by lucasrodes** [comment, marked as answer]
Hi @ariannecaritos-kinesso ,
Thanks for your message.

The UK is not reporting values for their vaccination metrics, so these won't show for recent dates. If you go back to August 2022, however, values for the UK should be available. This happens because Wales and Scotland don't report updated values, so the UK aggregate can't be reported. We report the UK nations individually, too, if that is of interest, you can check [here](https://ourworldindata.org/explorers/coronavirus-data-explorer?zoomToSelection=true&time=2022-05-07..latest&facet=none&pickerSort=desc&pickerMetric=new_cases_per_million&Metric=Vaccine+doses&Interval=7-day+rolling+average&Relative+to+Population=true&Color+by+test+positivity=false&country=England~Scotland~Northern+Ireland~Wales).

In the case of Australia, they have been reporting inconsistent data in the last month. Therefore updated values are unavailable.


I hope that this helps.

**Reply by ariannecaritos-kinesso** [reply]
Hi! thank you so much for your fast revert. Is this temporary or this is the way forward -- meaning both countries will not have aggregated and updated vaccination values for the next periods (next days, months, etc) ?

**Reply by lucasrodes** [reply]
After reaching out to the responsible for the UK dashboard, it seems that data on vaccinations will not be available anymore for the UK. As I mentioned, some UK nations have stopped reporting vaccination metrics.

As per Australia, we have switched to reported data by the WHO. So data is still available but with less frequency (last WHO report dates back to 2022-09-28)

**Reply by ariannecaritos-kinesso** [reply]
thank you so much! this is really helpful!
